Src/java/quick/src/main/resources/org/hl7/fhir/quickfhir-modelinfo-3.0.1.xml

```diff
@@ -0,0 +1,11543 @@
+        <element name="id" type="System.String" />
+        <element name="meta" type="QUICKFHIR.Meta" />
+        <element name="implicitRules" type="System.String" />
+        <element name="language" type="System.String" />
+        <element name="text" type="QUICKFHIR.Narrative" />
+        <element name="contained">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Resource" />
+        </element>
+        <element name="extension">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Extension" />
+        </element>
+        <element name="modifierExtension">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Extension" />
+        </element>
+        <element name="identifier" type="QUICKFHIR.Identifier" />
+        <element name="category" type="System.String" />
+        <element name="type" type="System.Concept" />
+        <element name="subject" type="QUICKFHIR.Reference" />
+        <element name="date" type="System.DateTime" />
+        <element name="reaction">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Reference" />
+        </element>
+        <element name="location" type="QUICKFHIR.Reference" />
+        <element name="seriousness" type="System.Concept" />
+        <element name="outcome" type="System.Concept" />
+        <element name="recorder" type="QUICKFHIR.Reference" />
+        <element name="eventParticipant" type="QUICKFHIR.Reference" />
+        <element name="description" type="System.String" />
+        <element name="suspectEntity">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.AdverseEvent.SuspectEntityComponent" />
+        </element>
+        <element name="subjectMedicalHistory">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Reference" />
+        </element>
+        <element name="referenceDocument">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Reference" />
+        </element>
+        <element name="study">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Reference" />
+        </element>
+    </typeInfo>
+    <typeInfo xsi:type="ClassInfo" baseType="QUICKFHIR.BackboneElement" name="AdverseEvent.SuspectEntityComponent">
+        <element name="id" type="System.String" />
+        <element name="extension">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Extension" />
+        </element>
+        <element name="modifierExtension">
+            <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Extension" />
+        </element>
+        <element name="instance" type="QUICKFHIR.Reference" />
+        <element name="causality" type="System.String" />
+    </typeInfo>
+    <typeInfo xsi:type="ClassInfo" baseType="QUICKFHIR.Quantity" name="Age" label="Age" primaryCodePath="code">
```

Comment:
Since GitHub messed this up, I'll reference Age below -- but note that I had several questions around this, and it applies to other types as well.  Line 2052:
```xml
<typeInfo xsi:type="ClassInfo" baseType="QUICKFHIR.Quantity" name="Age" label="Age" primaryCodePath="code">
    <element name="id" type="System.String" />
    <element name="extension">
        <typeSpecifier xsi:type="ListTypeSpecifier" elementType="QUICKFHIR.Extension" />
    </element>
    <element name="value" type="System.Decimal" />
    <element name="comparator" type="System.String" />
    <element name="unit" type="System.String" />
    <element name="system" type="System.String" />
    <element name="code" type="System.String" />
</typeInfo>
```

Reply:
And an example where `Age` is replaced by `System.Quantity` -- which removes the option of having a comparator and loses the context of what it actually is (e.g., it's impossible to tell from the model-info that it should be an age).  2nd choice in `Condition.onset`:
```xml
<element name="onset">
    <typeSpecifier xsi:type="ChoiceTypeSpecifier">
        <choice xsi:type="NamedTypeSpecifier" modelName="System" name="DateTime" />
        <choice xsi:type="NamedTypeSpecifier" modelName="System" name="Quantity" />
        <choice xsi:type="IntervalTypeSpecifier" pointType="System.DateTime" />
        <choice xsi:type="IntervalTypeSpecifier" pointType="System.Quantity" />
        <choice xsi:type="NamedTypeSpecifier" modelName="System" name="String" />
    </typeSpecifier>
</element>
```